spec/features/logging_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../web_helpers'
+
+feature 'Logging in:' do
+  include Helpers
+
+  scenario 'A user can log in' do
+    sign_up
+    log_in
+    expect(page).to have_content 'Hi, Yev Insomniak'
```

Comment:
The string 'Hi, Yev Insomniak' is a little bit mysterious. You can make it more obvious what you are testing if your sign_in method contains an argument 'Yev Insomniak'. That way, I could instantly see that the important thing here is that the name is being stored